### PR TITLE
Javascript javascript1 week3

### PR DESCRIPTION
**Pull request by vitalreddym**

### CODE REVIEW

File: javascript/javascript1/week3/main3.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function getTravelTime(object) {
+  let arrivalTime = (object.destinationDistance / object.speed) * 60;
+  let hours = Math.floor(arrivalTime / 60);
+  let minutes = Math.floor(arrivalTime % 60);
+  return hours + " hours and " + minutes + " minutes";
+}
```
Comment (Kristoffer88) [inline javascript/javascript1/week3/main3.js:39]: Overall good. But remember to look into the difference between let and const. If a variable does not change in it's lifetime, it better to use const.

File: javascript/javascript1/week3/main3.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+let lifeSpan = 80 * 365 * 24 * 60;
+let totalDuration = 0;
+  for (let i = 0; i < seriesDurations.length; i++) {
+    let duration = ((seriesDurations[i].days * 24 * 60 + seriesDurations[i].hours * 60 + seriesDurations[i].minutes) / lifeSpan) * 100;
+    console.log(seriesDurations[i].title + " took " + duration.toFixed(3) + "% of my life");
+    totalDuration += duration;
+  }
+  console.log("In total that is" + totalDuration.toFixed(2) + "% in my life");
+}
+  logOutSeriesText(seriesDurations);
```
Comment (Kristoffer88) [inline javascript/javascript1/week3/main3.js:81]: Look's good!

(look into const)

File: javascript/javascript1/week3/main3.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  const note = {
+    content: content,
+    id: id,
+  };
+notes.push(note);
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("Send an email", 3);
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
```
Comment (Kristoffer88) [inline javascript/javascript1/week3/main3.js:99]: Great work.

File: javascript/javascript1/week3/main3.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+function getNote(id) {
+  for(let i=0; i < getNote.length; i++) {
+    if (notes[i].id === id) {
+      return notes[i];
+    } else {
+      return "You entered wrong id";
+    }
+  }
+}
```
Comment (Kristoffer88) [inline javascript/javascript1/week3/main3.js:113]: Look's good. You can look into .find (notes.find(note=>note.id === 3))

File: javascript/javascript1/week3/main3.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+
+
+
+// Logout notes
+
+function logOutNotesFormatted() {
+  for (let i=0; i<notes.length; i++) {
+    console.log("The note with id: " + notes[i].id + ", has the following note text: " + notes[i].content);
+  }
```
Comment (Kristoffer88) [inline javascript/javascript1/week3/main3.js:124]: Great!

You can look into sending the note array in as a parameter. So you could have multiple note arrays

**Review by Kristoffer88 — APPROVED** [review]
Quick review before class

**Review by vitalreddym — COMMENTED** [review]
html

**Pull request merged**